### PR TITLE
CIV-5327 Hide R2 enhancement GA types

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5327

### Change description ###
Hide following GA types in R1

    "ListElementCode": "SET_ASIDE_JUDGEMENT",
   
    "ListElementCode": "VARY_ORDER",
   
    "ListElementCode": "ADJOURN_VACATE_HEARING",
    
    "ListElementCode": "UNLESS_ORDER",
    
    "ListElementCode": "OTHER",
   
    "ListElementCode": "VARY_JUDGEMENT",
    

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: bin/build-release-ccd-definition.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ elif [ ${environment} == local ]; then
 elif [ ${excludeNonProdFiles} == true ]; then
```
Comment (Azam-Hmcts) [inline bin/build-release-ccd-definition.sh:16]: It should be excluded in both if and else condition as we are not live yet. In my opinion name should be also like "....GAR2GAspec"  to continue with same naming pattern

File: ccd-definition/FixedLists/GeneralApplicationTypesGAspec-GAR2.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "ID": "GeneralApplicationTypesGAspec",
+    "ListElementCode": "SET_ASIDE_JUDGEMENT",
```
Comment (Azam-Hmcts) [inline ccd-definition/FixedLists/GeneralApplicationTypesGAspec-GAR2.json:4]: I would suggest to review the Display order as if both files are imported there will be conflict in displayorder

File: ccd-definition/FixedLists/GeneralApplicationTypesGAspec.json [hunk 3]
```diff
@@ -34,41 +34,5 @@
     "ListElementCode": "RELIEF_FROM_SANCTIONS",
     "ListElement": "Relief from sanctions",
     "DisplayOrder": 6
-  },
```
Comment (Azam-Hmcts) [inline ccd-definition/FixedLists/GeneralApplicationTypesGAspec.json:37]: In my opinion one file should contain all values which will be used in nonprod environment. Including new changes. One file should be wich contains only prod configuration. For us ATM all files containing name "...GAspec" are like nonProd files

File: ccd-definition/FixedLists/GeneralApplicationTypesGAspec-GAR2.json (hunk 2 above)
Comment (karthick-mohanasundaram-HMCTS) [reply ccd-definition/FixedLists/GeneralApplicationTypesGAspec-GAR2.json:4]: I guess we can ignore the -GAR2.json file in lower env too. @vasudevganesanhmcts your thoughts?

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Pull request closed without merging**